<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="centralWidgetLayout" columnminimumwidth="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="changeCamBtn">
        <property name="text">
         <string>change cam</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="startBtn">
        <property name="text">
         <string>start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="8">
       <widget class="QTextBrowser" name="messageTextBrowser">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="stopBtn">
        <property name="text">
         <string>stop</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="camIdSpnBx">
        <property name="minimum">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="houghParametersBtn">
        <property name="text">
         <string>HoughParameters</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="saveVideoButton">
        <property name="text">
         <string>save video as</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="resolutionCmbBx">
        <item>
         <property name="text">
          <string>640 x 480</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280 x 720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920 x 1080</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="recordVideoButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>record video</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../../../../.designer/icons/record.png</normaloff>
          <normalon>../../../../../.designer/icons/record_stop.png</normalon>../../../../../.designer/icons/record.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
